Library/Homebrew/extend/ENV/super.rb

```diff
@@ -176,6 +176,8 @@ def determine_library_paths
     PATH.new(
       keg_only_deps.map(&:opt_lib),
       HOMEBREW_PREFIX/"lib",
+      "#{MacOS.sdk_path}/usr/lib",
```

Comment:
Are you 100% sure adding this isn't going to affect any other builds in any other way?

Reply:
Actually regardless: only do this on `llvm_clang` being the compiler; it's not worth the risk.

Reply:
I think if CLT is installed we want `"/usr/lib"` not `""#{MacOS.sdk_path}/usr/lib"`

Reply:
Also, I'm not sure what happens if we do that and have also set
```
ENV["SDROOT"] = MacOS.sdk_path
```
in a formula, whether they're going to be competing with each other.

Reply:
@ilovezfs If both CLT and Xcode are installed, which should be preferred?

Reply:
CLT

Reply:
Isn't `MacOS.sdk_path` empty or `/` on CLT anyway?

Reply:
I just ended up splitting it into another if/else. Don't know if the nesting will be an issue.